test/e2e/volume_rm_test.go

```diff
@@ -100,7 +100,12 @@ var _ = Describe("Podman volume rm", func() {
 
 		session = podmanTest.Podman([]string{"volume", "rm", "myv"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).To(ExitWithError())
+		expect := "more than one result for volume name myv: volume already exists"
+		if podmanTest.DatabaseBackend == "boltdb" {
```

Comment:
`libpod/sqlite_state.go` line 2188 - change the %s to a %q and you can remove this conditional.

Reply:
I got my hands slapped last time I touched code.

Reply:
all I am saying I want a formal decision on what it should be %q vs %s rather than changing each differently, if we aim for consistency (which I agree with of course) we need define what consistency means first.

Reply:
Once all my ExitWithError PRs are merged, I intend to file an issue listing the inconsistencies (boltdb and remote) and calling for a decision.